spec/lib/fields/polymorphic_spec.rb

```diff
@@ -16,4 +17,23 @@
   end
 
   it { should_permit_param(:foo, for_attribute: :foo) }
+
+  describe "#display_associated_resource" do
+    it "displays through the dashboard based on the polymorphic class name" do
+      begin
+        Thing = Class.new
+        ThingDashboard = Class.new do
+          def display_resource(*)
+            :success
+          end
+        end
+
+        field = Administrate::Field::Polymorphic.new(:foo, Thing.new, :show)
+        display = field.display_associated_resource
+        expect(display).to eq :success
```

Comment:
Can you add a newline before this `expect`, to separate the test's execution phase from its expectation phase?

More at https://robots.thoughtbot.com/four-phase-test